.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
+about: Unexpected behavior? Tell us more
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+# Describe your issue
+
+(Tell us what the problem is in a few words)  
+*Ex: As an end user, when I access the console and see a service inventory, if attributes have too long values, I can't see any of the right buttons anymore*
```

Comment:
And then the example is not a very good one, because it does not use simple English sentences (I even think the sentence is incorrect). 

Same content, but a lot more concise and easier to read: "Long attributes will force the buttons off screen in the service inventory."